From 677f87af6bec31d896eb8c3754df2493bb4a1c28 Mon Sep 17 00:00:00 2001
From: Cody Peter Mello <cody.mello@joyent.com>
Date: Tue, 1 May 2018 20:44:42 +0000
Subject: [PATCH] TRITON-300 VMAPI "destroy" workflow timeout can leave behind
 undeletable networks (bump timeout) Reviewed by: Josh Wilsdon
 <jwilsdon@joyent.com> Approved by: Josh Wilsdon <jwilsdon@joyent.com>

---
 lib/workflows/destroy.js | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/lib/workflows/destroy.js b/lib/workflows/destroy.js
index 06a844d..4c39140 100644
--- a/lib/workflows/destroy.js
+++ b/lib/workflows/destroy.js
@@ -5,7 +5,7 @@
  */
 
 /*
- * Copyright (c) 2017, Joyent, Inc.
+ * Copyright (c) 2018, Joyent, Inc.
  */
 
 /*
@@ -20,7 +20,7 @@ var common = require('./job-common');
 var fabricCommon = require('./fabric-common');
 var nfsVolumes = require('./nfs-volumes');
 
-var VERSION = '7.1.3';
+var VERSION = '7.1.4';
 
 
 /*
@@ -208,7 +208,7 @@ var workflow = module.exports = {
         modules: { restify: 'restify' }
     }, {
         name: 'cnapi.wait_task',
-        timeout: 120,
+        timeout: 180,
         retry: 1,
         body: common.waitTask,
         modules: { sdcClients: 'sdc-clients' }
-- 
2.21.0

